src/sources/soundsourceffmpeg.cpp

```diff
@@ -1020,9 +1020,15 @@ ReadableSampleFrames SoundSourceFFmpeg::readSampleFramesClamped(
     CSAMPLE* pOutputSampleBuffer = writableSampleFrames.writableData();
 
     AVPacket avPacket;
```

Comment:
This does not fix the issue, that in future we must not rely on the size of AVPacket. 
It is intended to use av_packet_alloc() instead. 

see:

> Once removed, sizeof(AVPacket) will stop being a part of the public ABI.

Reply:
Thanks for the hint.